src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,141 @@
+package duke;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * The main class to deal with interactions with the user.
+ */
+public class Ui {
+
+    /**
+     * Displays the introduction to the user indicating that Duke is running.
+     */
+    public static void intro() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hi! I'm duke.Duke your friendly neighbourhood chat bot");
+        System.out.println("What can i do for you?");
+    }
+
+    /**
+     * Displays an indicator that the list is about to be shown
+     */
+    public static void tasks(TaskList list) {
+        System.out.println("        Here are the tasks in your list:");
+        int counter = 1;
+        for (Task t : list.getList()) {
+            System.out.println("        " + counter + "." + t.toString());
+            counter++;
+        }
+    }
+
+    public static String tasks(TaskList list, boolean isChat) {
+        String string1 = "Here are the tasks in your list:\n";
+        String string2 = "";
+        int counter = 1;
+        for (Task t : list.getList()) {
+            string2 += counter + "." + t.toString() + "\n";
+            counter++;
+        }
+        return string1 + string2;
+    }
+
+    /**
+     * Displays the error that was encountered as well as 2 lines of separation.
+     * @param e
+     */
+    public static void errorEncounter(Exception e) {
+        System.out.println("------------------------------------------------------");
+        System.out.println(e.getMessage());
+        System.out.println("------------------------------------------------------");
+    }
+
+
```

Comment:
There are 2 blank lines here, but one blank line between other methods. Try to keep it to one line between methods for consistency.